crates/assists/src/handlers/extract_struct_from_enum_variant.rs

```diff
@@ -401,17 +432,48 @@ fn f() {
 }
 "#,
             r#"
+//- /main.rs
 struct V{ pub i: i32, pub j: i32 }
 
 enum E {
     V(V)
 }
 mod foo;
 
+//- /foo.rs
+use crate::{E, V};
+fn f() {
+    let e = E::V(V { i: 9, j: 2 });
+}
 "#,
         )
     }
 
+    #[test]
+    fn test_extract_struct_record_nested_call_exp2() {
+        check_assist(
+            extract_struct_from_enum_variant,
+            r#"enum A { <|>One { a: u32, b: u32 } }
```

Comment:
```suggestion
            r#"
enum A { <|>One { a: u32, b: u32 } }
```

otherwise, the indent would be weird 

r=me otherwise